overlord/ifacestate/ifacemgr.go

```diff
@@ -174,6 +174,10 @@ func (m *InterfaceManager) StartUp() error {
 			return err
 		}
 	}
+	if snapdAppArmorServiceIsDisabled() {
```

Comment:
I think this should be part of apparmor backend Initialize() which called only when apparmor support has been found in the kernel.

Reply:
That was my initial implementation, but from there we don't have access to the `state` and therefore cannot emit warnings. Samuele in the chat suggested to have the interface manager do this check.